src/wagtail_localize_smartling/api/client.py

```diff
@@ -384,6 +385,49 @@ def add_file_to_job(self, *, job: "Job"):
             json=body,
         )
 
+    def add_html_context_to_job(self, *, job: "Job"):
```

Comment:
suggestion: add a docblock with a reference to https://api-reference.smartling.com/#tag/Context/operation/uploadAndMatchVisualContext